neptune_fastai/_version.py

```diff
@@ -1,3 +1,4 @@
+# pylint: skip-file
```

Comment:
What's wrong with this file?

Reply:
Maybe we should disable something in `pylintrc`?

Reply:
Currently I don't have an output of pylint but it also disabled in other integrations for ex.: https://github.com/neptune-ai/neptune-tensorflow-keras/blob/master/neptune_tensorflow_keras/_version.py